cce/cce.go

```diff
@@ -45,7 +46,9 @@ func (h *Handler) Get(c echo.Context) error {
 
 func (h *Handler) post(in *postRequest) {
 	h.Status = status.Collecting
-	h.SourceURL = fmt.Sprintf(h.SourceURL, in.Year)
+	if strings.Contains(h.SourceURL, "http") {
+		h.SourceURL = fmt.Sprintf(h.SourceURL, in.Year)
```

Comment:
Vai usar um padrão diferente quando for file://? Vai ser uma formatting URL quando for http e um path completo quando for file://

Independente, poderia documentar (adicionar comentários) isso, por favor?

Reply:
"Vai ser uma formatting URL quando for http e um path completo quando for file://"

A ideia é essa mesmo, porque o ano referente aos arquivos que queremos está presente na URL do TSE. Então se for passado um endereço com prefixo HTTP(S), esse ifzinho vai cuidar de concatenar o ano passado no corpo da requisição junto com o endereço em uma string pra que se faça o GET. 
A string não tendo prefixo HTTP(S) vamos assumir que é um path absoluto local

Reply:
Bem confuso ser de um jeito para uma configuração e outro diferente para outra, não acha? Uma boa aproveitar essa oportunidade para consolidar as configurações (documentar bem e etc). Acho que pode (e vai) confundir muito!

Reply:
Pensando bem, nada impede que tanto a URL quando o path dos arquivos contenham o ano 
Ex: 
- http://agencia.tse.jus.br/estatistica/sead/odsele/consulta_cand/consulta_cand_2016.zip
- file:///Users/abuarquemf/candidatos_2016.zip

Se padronizarmos isso esse if torna-se desnecessário

Reply:
E sobre documentação, acho que vale a pena de fato dedicar a isso assim que passarmos por isso

Reply:
Exato. Realmente não entendi se vai haver alguma ação aqui, dado meu comentário. Enfim .. marcando como resolvido para a gente prosseguir.